<!-- Image Processor Launch file -->
<launch>
        <node pkg="visnav" type="imagesave_node" name="imagesave_node" output="screen" respawn="true" />
        <!--      Loop rate for main landing node                   -->
        <param name="loop_rate"  type="double" value="100.0" />
        
        <!--				Input Topics							-->
        <param name="imagesaver_image_topic" type="str" value="/visnav/image_output" />
        
        <!--				Output Name							-->
        <param name="output_path"         type="str" value="/home/lsi/pictures/picture_" />


        <!-- 				Settings 							-->
        
        
        
        
</launch>
